saskatoon/sitebase/views.py

```diff
@@ -30,6 +31,36 @@ def get_context_data(self, **kwargs):
 
         return context
 
+    def dispatch(self, request, *args, **kwargs):
+        """Redirect new pickleaders"""
+
+        user = self.request.user
+
+        if user.is_authenticated:
+            # Retrieve only name fields from QuerySet of Groups
+            group_names = list(map(lambda group: group.name, user.role_groups))
```

Comment:
nit
```suggestion
            group_names = [g.name for g in user.role_groups]
```
might be `group_names = [g.name for g in user.role_groups.all()]`  i'm not sure